src/main/java/de/unistuttgart/towercrushbackend/service/websockets/GameService.java

```diff
@@ -21,6 +21,7 @@
 @Transactional
 public class GameService {
 
+    public static final String TEAM_A = "teamA";
```

Comment:
I am deeply disturbed.
It does not seem like a good idea to **hardcode** teams.
I am only going to approve this PR because it isn't the one introducing it.
But please use normal object oriented programming instead of… that.